ntbs-service/deployments/live.yml

```diff
@@ -122,6 +122,10 @@ spec:
             # fallback to appSettings.
             - name: ScheduledJobsConfig__ReportingDataRefreshJobCron
               value: "30 07 * * *"
+            # We are disabling this job temporarily while users are being asked to make changes to data
+            # that will be in the annual report and we decide on a new timeframe for notification closure.
```

Comment:
I'm not sure this comment is strictly accurate - we're not really sure yet whether this will be a temporary or permanent change.

I think the most important thing to do here is to include the Jira ticket number in the comment, so that someone can look it up later.